terraform/variables.tf

```diff
@@ -0,0 +1,13 @@
+variable "domain" {
+  description = "The domain where Rabble Rouser is deployed."
+}
+
+variable "region" {
+  default = "ap-southeast-2"
+}
+
+variable "redirector_lambda_name" {
+  default = "redirector"
+}
```

Comment:
I would probably hardcode this in `redirector.tf`, and then in `api_gateway.tf` you can refer to it as `"{aws_lambda_function.redirector.function_name}"`.